From: Emmanuel Arias <eamanu@yaerobi.com>
Date: Mon, 29 Jun 2020 11:08:16 -0300
Subject: Update change docs theme patch

---
 doc/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/conf.py b/doc/conf.py
index 3ab5c2e..98ad14d 100644
--- a/doc/conf.py
+++ b/doc/conf.py
@@ -85,7 +85,7 @@ exclude_patterns = ['_build']
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'pygments14'
+html_theme = 'default'
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
